.github/workflows/tflint.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint
+on:
+  push:
+    branches: [ "prod", "dev", "staging" ]
+  pull_request: 
+    branches: [ "prod" ]
+
+jobs:
+  tflint:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
macos and windows are not necessarily, Ubuntu is sufficient.

Reply:
I'll make this change shortly, thanks.